src/aashay/staticNonStaticMethods/StaticNonStaticReturnTypes.java

```diff
@@ -0,0 +1,63 @@
+package aashay.staticNonStaticMethods;
+
+		
+	
+public class StaticNonStaticReturnTypes {
+	
+	double sum(double num1, double num2, double num3) {
+		double sum = num1+num2+num3;
+		return sum;
+		
+	}
+	double average(double num1, double num2, double num3) {
+		double addition=  sum(num1,num2,num3);
+		double average = addition/3;
+		return average;
+	}
+	static boolean isAnswerAboveExpecation(double avrge) {
+		if(avrge >50 ) {
+			return true ;
+		}else
+			return false;
+	}
+	static boolean booleanisEligible(double avrge) {
+		if(avrge >50 || (avrge%2 ==0)) {
```

Comment:
we need to use divisible function not modular.
if value is more than 50 or divisible by 2. Eligibility will be printed in main method.